### ISSUE TITLE
Choice of License

### ISSUE THREAD

**Issue by voremargot** [UBC-MDS/strapR#50]
Our package is being distributed under the MIT License. 

When choosing a software license, we knew we wanted our package to be easily accessible and truly open source. We do not want to hamper our package’s ability to help improve the functionality of someone else’s code or project by adding complex layers of usage. Thus, we have chosen to use a permissive license rather than a copyleft license.

The two permissive licenses we explored were MIT License and the BSD License. While the two licenses are very similar, the MIT license is more explicit in how you can use our work, such as merge it, publish it, and sell it. While this is implied in the BSD License, room for interpretation exists. MIT also specifically addresses the software and associated documentation whereas BSD only addresses the source code itself. Overall, MIT is widely used in the community and is explicit in its permissions thus we selected it as our package’s license.

**Comment by gauthampughazhendhi** [comment]
@voremargot as per our discussion earlier, I also agree with the selection of the MIT license since it is developer-friendly and allows more people to use our package without being restricted by licensing terms. For example, a GPL license requires the user who uses our package to also follow the GPL license for their product. The MIT license is widely adopted and used by many popular packages such as Tidyverse and so on. So, I agree with this choice of license. Thank you!

**Comment by BooleanJulien** [comment]
I am also pro MIT. To add to the discussion, a bit of online research (albeit, not from peer reviewed sources) suggested that MIT is the most common open source license used for software. This is a useful trait since many potential users will be familiar with the content of the license. Moreover, the license is classified as _permissive_ which is a useful characteristic for a project such as this, where we are more than happy for anybody to derive other useful packages and work from our code.  

Sources:
https://blog.containerize.com/2021/09/23/top-5-most-popular-osi-approved-open-source-licenses-of-2021/
https://medium.com/nationwide-technology/a-short-guide-to-open-source-licenses-cf5b1c329edd

**Comment by zackt113** [comment]
Yeah,  I totally agree with you guys. I think MIT license is one of the most permissible licenses. Especially for our project, we could modify and edit codes without any issues by using MIT license. Also, it is relatively short and simple to write compared to other open-source licenses. It is good for both commercial use and personal use. I would say it is better f we use MIT license! Thank you!
